src/Bicep.Cli/InvocationContext.cs

```diff
@@ -36,9 +35,7 @@ public class InvocationContext
 
         public TextWriter ErrorWriter { get; }
 
-        public EmitterSettings EmitterSettings => new EmitterSettings(Features);
-
-        public IFeatureProvider Features { get; }
+        public IFeatureProviderManager? FeatureProviderManager { get; }
```

Comment:
It doesn't seem like we use this property anywhere other than in test code. Do we need to use it?

Reply:
We didn't have a separate mechanism to turn features on or off for tests, but the last commit removes this and uses CLI arguments instead.